source/myukfast/accounts.md

```diff
@@ -0,0 +1,27 @@
+# Accounts
+
+```eval_rst
+   .. title:: MyUKFast | Accounts
+   .. meta::
+      :title: MyUKFast | Managing Contacts and Permissions | UKFast Documentation
+      :description: How to perform various Accounts related activities in MyUKFast
```

Comment:
this title is a better one, and good for search if it was
Managing Contacts and Permissions in MyUKFast